website/templates/project/modal_add_contributor.mako

```diff
@@ -64,8 +59,15 @@
                                         <td class="p-r-sm osf-icon-td" >
                                             <a
                                                     class="btn btn-success contrib-button btn-mini"
-                                                    data-bind="click:$root.add, tooltip: {title: 'Add contributor'}"
-                                                ><i class="fa fa-plus"></i></a>
+                                                    data-bind="visible: !contributor.added,
+                                                               click:$root.add,
+                                                               tooltip: {title: 'Add contributor'}"
+                                                ><i class="fa fa-fw fa-plus"></i></a>
+                                            <a
+                                                class="btn btn-default contrib-button .disabled btn-mini"
```

Comment:
I don't think the disabled class needs a period in front of it. Also I just remembered the tooltips won't show if the button is disabled!